### PR TITLE
contrib/gamescope: update to 3.14.1

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

**Comment by JamiKettunen** [comment]
`pkgname`s are e.g. `wlroots0.17` but commit messages have `wlroots-0.17` (same for 0.16)

**Comment by nekopsykose** [comment]
true, thanks

File: contrib/wlroots0.16/template.py [hunk 1]
```diff
@@ -27,18 +27,19 @@
     "xcb-util-renderutil-devel",
     "xcb-util-wm-devel",
 ]
+replaces = ["wlroots"]
```
Comment (q66) [inline contrib/wlroots0.16/template.py:30]: this is nowhere near enough (e.g. people who currently have wlroots-devel installed explicitly in world will have leftover files)

i'd rather have one of the versions (probably the newer one) provide unversioned names virtually for both the main package and the devel package, which will take care of a proper transition

File: contrib/wlroots0.16/template.py [hunk 2]
```diff
@@ -27,18 +27,19 @@
...
 pkgdesc = "Modular Wayland compositor library"
```
Comment (q66) [inline contrib/wlroots0.16/template.py:31]: pkgdesc for both should say which branch it is (i.e. `(0.16.x)` at the end)

File: contrib/wlroots0.16/template.py (hunk 1 above)
Comment (nekopsykose) [reply contrib/wlroots0.16/template.py]: sure, added. the replaces= also wasn't necessary since the files don't conflict with the old ones

**Pull request merged**